chart/dapr-shared/templates/helmhooks.yaml

```diff
@@ -54,6 +54,7 @@ metadata:
     app.kubernetes.io/instance: {{ .Release.Name | quote }}
     app.kubernetes.io/version: {{ .Chart.AppVersion }}
     helm.sh/chart: "{{ .Chart.Name }}-{{ .Chart.Version }}"
+    sidecar.istio.io/inject: "false"
```

Comment:
This labels appears to be specific to those using Istio. To make it more generic consider introducing `.Values.podLabels`, similar to the already present `.Values.podAnnotations`. Or, if this need to be specific, to the delete-cm-task, add a dedicated value in values.yaml.